subscription/src/main/java/org/killbill/billing/subscription/api/transfer/DefaultSubscriptionBaseTransferApi.java

```diff
@@ -120,6 +120,10 @@ private SubscriptionBaseEvent createEvent(final boolean firstEvent, final Existi
                 newEvent = BCDEventData.createBCDEvent(subscription, effectiveDate, existingEvent.getBillCycleDayLocal());
                 break;
 
+            case QUANTITY_CHANGE:
+                // TODO
```

Comment:
Maybe throw unsupported?

Reply:
Just to be clear: With the current code we allow the transfer, but we lose the `quantity` as it is not 'ported' with the subscription. If I understand correctly you are proposing the prevent the transfer from happening in such case to make it clear?

Ideally, I would like to fix it in as part of #1627, which I kept opened.

Reply:
Followed your recommendation for now in https://github.com/killbill/killbill/pull/1805/commits/e375c3c360a421c90cebff5e262d5d9f7ca9c490